tests/hpc/slurm_master.pm

```diff
@@ -5,19 +5,23 @@
 
 # Summary: Slurm master node
 #    This test is setting up slurm master node and runs tests depending
-#    on the slurm cluster configuration
+#    on the slurm cluster configuration.
+#    SLURM_VERSION enables installation of the particular versioned Slurm.
 # Maintainer: Kernel QE <kernel-qa@suse.de>
 
 use Mojo::Base qw(hpcbase hpc::configs), -signatures;
 use testapi;
 use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
 use lockapi;
 use utils;
+use hpc::utils 'get_slurm_version';
 use version_utils 'is_sle';
 use Utils::Logging 'export_logs_basic';
 
 our @all_tests_results;
 
+our $slurm_pkg = get_slurm_version();
```

Comment:
There is no need to have it as global variable. Use it in the code like we usually do.

Reply:
it is not needed for most of the modules. but it does for `slurm_master`.  The variable is used on `t08_basic`. i think it makes sense to not touch anything here and just use the variable as public for this case.

Reply:
It will be better to update `t08_basic` code with one line. I will be at one place much clear where is variable coming from.